code/controllers/subsystem/air.dm

```diff
@@ -615,7 +615,7 @@ SUBSYSTEM_DEF(air)
 // In an ideal world, we would have absolutely zero active turfs 99.99% of the time, but that's not the case. `log_mapping()` during world initialize triggers a CI fail.
 #ifdef UNIT_TESTS
 	return
-#endif
+#else
```

Comment:
why?

Reply:
oh for linting unreachable code. yep fair enough

Reply:
>[fbe39c9](https://github.com/tgstation/tgstation/pull/75570/commits/fbe39c9aecbf77f9035317d41ac640ffc2993c50)
`Suppress unreachable code lints with UNIT_TESTS defined`